cmd/swarm-smoke/upload_and_sync.go

```diff
@@ -178,12 +178,26 @@ func trackChunks(testData []byte, submitMetrics bool) error {
 // getChunksBitVectorFromHost returns a bit vector of presence for a given slice of chunks from a given host
 func getChunksBitVectorFromHost(client *rpc.Client, addrs []storage.Address) (string, error) {
 	var hostChunks string
-
-	err := client.Call(&hostChunks, "bzz_has", addrs)
-	if err != nil {
-		return "", err
+	lastPage := false
```

Comment:
1. Please extract 7500 as a `limit` variable.

2. I think it would be simpler if we have fewer variable to iterate the pages of the `addrs` slice, you can get rid of `lastPage`, for example:

```
	addrs := []int{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10}

	limit := 4

	for len(addrs) > 0 {
		// get current page size, so that we avoid a slice out of bounds on the last page
		pagesize := limit
		if cap(addrs) < limit {
		  pagesize = cap(addrs)
		}

		// process
		fmt.Println("current page:", addrs[:pagesize])

		addrs = addrs[pagesize:]
	}
```

Reply:
nice catch @nonsense